docs/source/quark_script.rst

```diff
@@ -207,6 +207,13 @@ methodInstance.getArguments(none)
 - **Description**: Get arguments from method.
 - **params**: none
 - **return**: python list containing arguments.
+  
+methodInstance.findSuperclassHierarchy(none)
+==============================
```

Comment:
According to [the reStructuredText spec.](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections), the underline of the title should be as long as the title.

For example,
```
methodInstance.findSuperclassHierarchy(none)
=========================================
```